docs/GETTING_STARTED.md

```diff
@@ -79,6 +79,14 @@ You will get two json files `mask_rcnn_test-dev_results.bbox.json` and `mask_rcn
 
 The generated png and txt would be under `./mask_rcnn_cityscapes_test_results` directory.
 
+6. Test Mask R-CNN with 8 GPUs, and evaluate the bbox and mask AP.
```

Comment:
- This example may be put between 3 and 4
- `evaluate the **classwise** bbox and mask AP`